app/logic/Mail/AutoConfig/manualConfig.ts

```diff
@@ -16,7 +16,7 @@ export function makeManualConfig(emailAddress: string, password: string): MailAc
   config.authMethod = AuthMethod.Password;
   config.source = "manual";
 
-  let outgoing = new SMTPAccount();
+  let outgoing = newAccountForProtocol("smtp") as any as SMTPAccount;
```

Comment:
Was account setup broken because it wasn't saving the account or something? (I'm worried that these changes relate to the "FOREIGN KEY constraint failed" errors I was seeing.)

(The `as any` shouldn't be necessary, but I've had similar problems before, so I'll take your word for it.)